### PR TITLE
Fixes #12044: Add a script to rewrite a technique from command line

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://www.rudder-project.org/redmine/issues/12044

### CODE REVIEW

**Comment by peckpeck** [comment]
You should include this code in the ncf binary

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
OK, merging this PR

File: ncf [hunk 1]
```diff
@@ -53,10 +50,52 @@ def init(args):
...
 if __name__ == '__main__':
+
   if len(sys.argv) <= 1:
     usage()
     exit(1)
 
   if sys.argv[1] == "init":
     init(sys.argv[2:])
+  elif sys.argv[1] == "write_all_techniques":
+    write_all_techniques(sys.argv[2], None)
```
Comment (ncharles) [inline ncf]: this should be write_techniques

**Pull request merged**